appengine/taskqueue/counter/queue.yaml

```diff
@@ -0,0 +1,3 @@
+queue:
```

Comment:
Looks like the first snippit in https://cloud.google.com/appengine/docs/python/config/queue uses it